frontend/packages/dev-console/src/components/import/route/SecureRoute.tsx

```diff
@@ -15,20 +15,19 @@ const SecureRoute: React.FC = () => {
     },
   } = useFormikContext<FormikValues>();
   return (
-    <FormGroup>
-      <ControlLabel>Security</ControlLabel>
+    <React.Fragment>
       <CheckboxField
         name="route.secure"
         label="Secure Route"
+        formLabel="Security"
         helpText="Routes can be secured using several TLS termination types for serving certificates."
       />
       {secure && (
-        <div>
+        <React.Fragment>
```

Comment:
Why do you need these nested Fragments? What does it give you?

Reply:
Because PF4 forms use a grid layout for spacing and only direct children of the `pf-c-form` class get the spacing. Here we need the nodes to be direct children so removing the intermediate node is necessary.